frontend/src/components/comments/comment_index_item.jsx

```diff
@@ -9,20 +9,19 @@ class CommentIndexItem extends React.Component {
   }
 
   render() {
+    console.log(this.props.comment)
```

Comment:
What's this here for? Should probably remove

Reply:
You're right. Can I do that from here?

Reply:
I think so